java/client/src/org/openqa/selenium/print/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("//java:defs.bzl", "java_library")
+
+java_library(
+    name = "print",
+    srcs = glob(["**/*.java"]),
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        "//java/client/src/org/openqa/selenium:core",
```

Comment:
I'd suggest that this package needs to be exported by `core`, so you may just want to add `print/*.java` to `core`'s definition.